pkg/operation/seed/seed.go

```diff
@@ -438,6 +438,12 @@ func BootstrapCluster(seed *Seed, secrets map[string]*corev1.Secret, imageVector
 			"enabled":        vpaEnabled,
 			"podAnnotations": vpaPodAnnotations,
 		},
+		"global-network-policies": map[string]interface{}{
+			// TODO (mvladev): Move the Provider specific metadata IP
```

Comment:
Please don't do that so that we can keep the seed controller cloud provider agnostic.